fix: update test and coverage (#351)
screwdriver-cd/screwdriver-cd.github.io: docs/user-guide/metadata.md

### Coverage and Test Results

> You can populate coverage and test results and set their url to build artifact on build page from a Screwdriver build through using metadata. Screwdriver UI will read from `tests.coverage`, `tests.results`, `tests.coverageUrl` and `tests.resultsUrl` in metadata and display/set them accordingly.

Reviewer: ```suggestion
You can populate coverage results and test results, along with their respective URLs, on the build page by setting metadata during a Screwdriver build. The Screwdriver UI will read from `tests.coverage`, `tests.results`, `tests.coverageUrl`, and `tests.resultsUrl` metadata and display/set them accordingly.
```

      - set-coverage-and-test-results: |
          meta set tests.coverage 100 # this should be the coverage percentage number
          meta set tests.results 10/10 # this should be `pass_tests_number/total_tests_number`

>           meta set tests.results /test/testReport.html # this should be a relative path to a build artifact

Reviewer: ```suggestion
          meta set tests.resultsUrl /test/testReport.html # this should be a relative path to a build artifact
```

>           meta set tests.results /test/coverageReport.html # this should be a relative path to a build artifact

Reviewer: ```suggestion
          meta set tests.coverageUrl /test/coverageReport.html # this should be a relative path to a build artifact
```

Author: changed
